### PR TITLE
Style floating header

### LINKED ISSUE #115: "DIRECT" label blurred on Linux after scorlling
The "DIRECT" label in the channel list get blurred after scrolling on Linux.

### PR DESCRIPTION
**Pull request by jperedadnr**
Fixes #115

### CODE REVIEW

File: App/src/main/java/com/gluonhq/chat/views/ChannelPresenter.java [hunk 1]
```diff
@@ -35,7 +40,28 @@ public void initialize() {
         }));
         channelList.setItems(channelFilteredList);
         channelList.setCellFactory(param -> new ChannelCell());
-        channelList.setHeadersFunction(param -> param.isDirect() ? "DIRECT" : "CHANNELS");
+        channelList.setHeadersFunction(param -> param.isDirect() ? "DIRECT" : "COMMON");
```
Comment (johanvos) [inline App/src/main/java/com/gluonhq/chat/views/ChannelPresenter.java:43]: better "GROUPS" instead of COMMON

Comment (jperedadnr) [reply App/src/main/java/com/gluonhq/chat/views/ChannelPresenter.java]: Fixed

File: App/src/main/java/com/gluonhq/chat/views/ChannelPresenter.java [hunk 2]
```diff
@@ -35,7 +40,28 @@ public void initialize() {
         }));
         channelList.setItems(channelFilteredList);
         channelList.setCellFactory(param -> new ChannelCell());
-        channelList.setHeadersFunction(param -> param.isDirect() ? "DIRECT" : "CHANNELS");
+        channelList.setHeadersFunction(param -> param.isDirect() ? "DIRECT" : "GROUP");
```
Comment (eugener) [inline App/src/main/java/com/gluonhq/chat/views/ChannelPresenter.java:43]: Should those be loaded from properties too? For future i18n capability.

File: App/src/main/java/com/gluonhq/chat/views/ChannelPresenter.java [hunk 3]
```diff
@@ -35,7 +40,28 @@ public void initialize() {
...
+                box.getStyleClass().add("header-box");
+            }
+
+            @Override
+            public void updateItem(Channel item, boolean empty) {
+                super.updateItem(item, empty);
+                if (item == null || empty) {
+                    setGraphic(null);
+                } else {
+                    label.setText(resources.getString(item.isDirect() ? "channels.category.direct" : "channels.category.common"));
```
Comment (eugener) [inline App/src/main/java/com/gluonhq/chat/views/ChannelPresenter.java:60]: For consistency, change `common` to `group`

**Review by eugener — APPROVED** [review]

**Pull request merged**